crates/types/src/services/executor.rs

```diff
@@ -358,3 +358,20 @@ impl From<ValidityError> for TransactionValidityError {
         Self::Validation(CheckError::Validity(e))
     }
 }
+
+/// Error type for the upgradable (wasm) executor.
+#[cfg(feature = "wasm-executor")]
```

Comment:
Maybe it is better to move it to the `fuel-core-upgradable-executor` and avoid adding a new feature=)

Reply:
Done in ed9169d30